### ISSUE TITLE
Timeout please help

### ISSUE THREAD

**Issue by chikcken** [MSCMP/MSCMP#28]
Help me me and my friend getting timeout when joining
we deleted our save files and disabled mods.......
my friend still have modloader but he disabled it
any solution? please?

**Comment by DarkKostas** [comment]
Timeout is a well know "bug" that happens when the player connecting, takes too long to join the game. Try swap places and make him the host instead. If it still doesn't work, you can't do anything for now.

**Comment by chikcken** [comment]
alright i tried to join him before sending this and he tried to join me nothing works..... i understand this that i can't do anything but thanks for the information.....

**Comment by DarkKostas** [comment]
Timeout shouldn't normally count when you're joining but only after player successfully joins(or at least increase the timer when joining). It's something on the to-do list.

**Comment by CurtisVL** [comment]
I believe the timeout time was raised at some point, so this issue might now be resolved?
If it's still an issue in the latest dev build (That you can download from our Discord server) please re-open a new issue! 👍
